definitions/pipelines/pvacseq.cwl

```diff
@@ -180,7 +180,7 @@ steps:
             fasta_size: fasta_size
             downstream_sequence_length: downstream_sequence_length
             exclude_nas: exclude_nas
-            phased_proximal_variants_file: phased_proximal_variants_file
+            phased_proximal_variants_vcf: phased_proximal_variants_file
```

Comment:
Should we change the input name for this pipeline, too?

(How did this mismatch slip through before?  Travis should notice this!)